Uppercase table names breaks sync

### Describe the bug

In postgres the table "people" and "PEOPLE" can coexist, making the sync process to break

### To Reproduce

1) using the sample dataset run
```
CREATE TABLE "public"."PEOPLE"
(id int)
```
2) add the database
3) now the people table can't be used, as Metabase confuses both

![Image](https://github.com/user-attachments/assets/553c8cab-9cc0-4eae-9660-db9ea381798e)

![Image](https://github.com/user-attachments/assets/433de6e5-4b1a-4b36-9873-7345b4fa8268)

![Image](https://github.com/user-attachments/assets/c61bccf9-0873-47a6-ae1e-0e565d6fa65f)

### Expected behavior

We should identify that there are tables with the same name and sync both

### Logs

```
2025-04-20 23:16:08,807 INFO sync.util :: STARTING: step ''sync-tables'' for postgres Database 13371341 ''data''
2025-04-20 23:16:08,813 INFO sync-metadata.tables :: Found new table: Table  ''public.products''
2025-04-20 23:16:08,815 INFO sync-metadata.tables :: Found new table: Table  ''public.PEOPLE''
2025-04-20 23:16:08,817 INFO sync-metadata.tables :: Found new table: Table  ''public.reviews''
2025-04-20 23:16:08,820 INFO sync-metadata.tables :: Found new table: Table  ''public.orders''
2025-04-20 23:16:08,823 INFO sync-metadata.tables :: Found new table: Table  ''public.invoices''
2025-04-20 23:16:08,825 INFO sync-metadata.tables :: Found new table: Table  ''public.people''
2025-04-20 23:16:08,827 INFO sync-metadata.tables :: Found new table: Table  ''public.analytic_events''
2025-04-20 23:16:08,828 INFO sync-metadata.tables :: Found new table: Table  ''public.accounts''
2025-04-20 23:16:08,830 INFO sync-metadata.tables :: Found new table: Table  ''public.feedback''
2025-04-20 23:16:08,992 INFO sync-metadata.tables :: Updating table metadata for Table 203 ''public.accounts''
2025-04-20 23:16:08,994 INFO sync-metadata.tables :: :estimated_row_count of Table 203 ''public.accounts'' changed from null to 2495
2025-04-20 23:16:09,000 INFO sync-metadata.tables :: Updating table metadata for Table 204 ''public.feedback''
2025-04-20 23:16:09,001 INFO sync-metadata.tables :: :estimated_row_count of Table 204 ''public.feedback'' changed from null to 642
2025-04-20 23:16:09,006 INFO sync-metadata.tables :: Updating table metadata for Table 196 ''public.products''
2025-04-20 23:16:09,009 INFO sync-metadata.tables :: :estimated_row_count of Table 196 ''public.products'' changed from null to 200
2025-04-20 23:16:09,015 INFO sync-metadata.tables :: Updating table metadata for Table 197 ''public.PEOPLE''
2025-04-20 23:16:09,017 INFO sync-metadata.tables :: :estimated_row_count of Table 197 ''public.PEOPLE'' changed from null to 0
2025-04-20 23:16:09,022 INFO sync-metadata.tables :: Updating table metadata for Table 198 ''public.reviews''
2025-04-20 23:16:09,024 INFO sync-metadata.tables :: :estimated_row_count of Table 198 ''public.reviews'' changed from null to 1112
2025-04-20 23:16:09,031 INFO sync-metadata.tables :: Updating table metadata for Table 200 ''public.invoices''
2025-04-20 23:16:09,033 INFO sync-metadata.tables :: :estimated_row_count of Table 200 ''public.invoices'' changed from null to 13856
2025-04-20 23:16:09,038 INFO sync-metadata.tables :: Updating table metadata for Table 199 ''public.orders''
2025-04-20 23:16:09,040 INFO sync-metadata.tables :: :estimated_row_count of Table 199 ''public.orders'' changed from null to 18760
2025-04-20 23:16:09,046 INFO sync-metadata.tables :: Updating table metadata for Table 201 ''public.people''
2025-04-20 23:16:09,048 INFO sync-metadata.tables :: :estimated_row_count of Table 201 ''public.people'' changed from null to 2500
2025-04-20 23:16:09,053 INFO sync-metadata.tables :: Updating table metadata for Table 202 ''public.analytic_events''
2025-04-20 23:16:09,055 INFO sync-metadata.tables :: :estimated_row_count of Table 202 ''public.analytic_events'' changed from null to 37169
2025-04-20 23:16:09,064 INFO sync.util :: FINISHED: step ''sync-tables'' for postgres Database 13371341 ''data'' (257.1 ms)
2025-04-20 23:16:09,066 INFO sync.util :: STARTING: step ''sync-fields'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,069 DEBUG sync.fetch-metadata :: Using `describe-fields` (fast sync) to fetch fields metadata.
2025-04-20 23:16:09,189 INFO fields.sync-metadata :: Database type of Table 197 ''public.PEOPLE'' Field 'id' has changed from 'int4' to 'int8'.
2025-04-20 23:16:09,191 INFO fields.sync-metadata :: Base type of Table 197 ''public.PEOPLE'' Field 'id' has changed from ':type/Integer' to ':type/BigInteger'. This field will be refingerprinted and analyzed.
2025-04-20 23:16:09,193 INFO fields.sync-metadata :: Semantic type of Table 197 ''public.PEOPLE'' Field 'id' has changed from 'null' to ':type/PK'.
2025-04-20 23:16:09,195 INFO fields.sync-metadata :: Database required of Table 197 ''public.PEOPLE'' Field 'id' has changed from 'false' to 'true'.
2025-04-20 23:16:09,230 INFO sync.util :: FINISHED: step ''sync-fields'' for postgres Database 13371341 ''data'' (164.4 ms)
2025-04-20 23:16:09,233 INFO sync.util :: STARTING: step ''sync-fks'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,244 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.analytic_events'' Field  ''account_id'' -> Table  ''public.analytic_events'' Field  ''id''
2025-04-20 23:16:09,248 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.feedback'' Field  ''account_id'' -> Table  ''public.feedback'' Field  ''id''
2025-04-20 23:16:09,251 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.invoices'' Field  ''account_id'' -> Table  ''public.invoices'' Field  ''id''
2025-04-20 23:16:09,254 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.orders'' Field  ''product_id'' -> Table  ''public.orders'' Field  ''id''
2025-04-20 23:16:09,258 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.orders'' Field  ''user_id'' -> Table  ''public.orders'' Field  ''id''
2025-04-20 23:16:09,261 INFO sync-metadata.fks :: Marking foreign key from Table  ''public.reviews'' Field  ''product_id'' -> Table  ''public.reviews'' Field  ''id''
2025-04-20 23:16:09,274 INFO sync.util :: FINISHED: step ''sync-fks'' for postgres Database 13371341 ''data'' (41.6 ms)
2025-04-20 23:16:09,277 INFO sync.util :: STARTING: step ''sync-indexes'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,289 INFO sync-metadata.indexes :: Marking Field 2929 as indexed
2025-04-20 23:16:09,291 INFO sync-metadata.indexes :: Marking Field 2913 as indexed
2025-04-20 23:16:09,293 INFO sync-metadata.indexes :: Marking Field 2894 as indexed
2025-04-20 23:16:09,295 INFO sync-metadata.indexes :: Marking Field 2956 as indexed
2025-04-20 23:16:09,297 INFO sync-metadata.indexes :: Marking Field 2908 as indexed
2025-04-20 23:16:09,298 INFO sync-metadata.indexes :: Marking Field 2919 as indexed
2025-04-20 23:16:09,300 INFO sync-metadata.indexes :: Marking Field 2946 as indexed
2025-04-20 23:16:09,310 INFO sync.util :: FINISHED: step ''sync-indexes'' for postgres Database 13371341 ''data'' (33.2 ms)
2025-04-20 23:16:09,312 INFO sync.util :: STARTING: step ''sync-metabase-metadata'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,317 INFO sync.util :: FINISHED: step ''sync-metabase-metadata'' for postgres Database 13371341 ''data'' (4.7 ms)
2025-04-20 23:16:09,319 INFO sync.util :: STARTING: step ''sync-table-privileges'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,329 INFO sync.util :: FINISHED: step ''sync-table-privileges'' for postgres Database 13371341 ''data'' (9.3 ms)
2025-04-20 23:16:09,344 INFO sync.util :: FINISHED: Sync metadata for postgres Database 13371341 ''data'' (924.7 ms)
2025-04-20 23:16:09,347 DEBUG sync.util :: Sync operations in flight: {:analyze #{13371341}, :sync #{13371341}}
2025-04-20 23:16:09,349 INFO sync.util :: STARTING: Analyze data for postgres Database 13371341 ''data''
2025-04-20 23:16:09,353 INFO sync.util :: STARTING: step ''fingerprint-fields'' for postgres Database 13371341 ''data''
2025-04-20 23:16:09,364 INFO analyze.fingerprint :: Fingerprinting 15 fields in table Table 203 ''public.accounts''
2025-04-20 23:16:09,423 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2887 ''latitude''
2025-04-20 23:16:09,431 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2888 ''country''
2025-04-20 23:16:09,436 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2889 ''email''
2025-04-20 23:16:09,440 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2890 ''trial_ends_at''
2025-04-20 23:16:09,446 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2891 ''longitude''
2025-04-20 23:16:09,450 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2892 ''trial_converted''
2025-04-20 23:16:09,504 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2893 ''plan''
2025-04-20 23:16:09,511 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2895 ''seats''
2025-04-20 23:16:09,516 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2896 ''last_name''
2025-04-20 23:16:09,520 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2897 ''source''
2025-04-20 23:16:09,524 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2898 ''legacy_plan''
2025-04-20 23:16:09,529 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2899 ''canceled_at''
2025-04-20 23:16:09,533 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2900 ''first_name''
2025-04-20 23:16:09,537 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2901 ''active_subscription''
2025-04-20 23:16:09,541 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2902 ''created_at''
2025-04-20 23:16:09,547 INFO analyze.fingerprint :: Fingerprinting 6 fields in table Table 204 ''public.feedback''
2025-04-20 23:16:09,626 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2909 ''email''
2025-04-20 23:16:09,631 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2910 ''rating_mapped''
2025-04-20 23:16:09,636 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2911 ''rating''
2025-04-20 23:16:09,640 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2912 ''date_received''
2025-04-20 23:16:09,646 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2914 ''body''
2025-04-20 23:16:09,704 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2915 ''account_id''
2025-04-20 23:16:09,709 INFO sync.analyze :: fingerprint-fields Analyzed [*****·············································]    11% Table 196 ''public.products''
2025-04-20 23:16:09,714 INFO analyze.fingerprint :: Fingerprinting 7 fields in table Table 196 ''public.products''
2025-04-20 23:16:09,739 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2944 ''category''
2025-04-20 23:16:09,745 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2945 ''ean''
2025-04-20 23:16:09,749 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2947 ''created_at''
2025-04-20 23:16:09,752 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2948 ''vendor''
2025-04-20 23:16:09,756 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2949 ''title''
2025-04-20 23:16:09,761 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2950 ''rating''
2025-04-20 23:16:09,765 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2951 ''price''
2025-04-20 23:16:09,770 INFO analyze.fingerprint :: Fingerprinting 13 fields in table Table 197 ''public.PEOPLE''
2025-04-20 23:16:09,816 WARN sync.util :: Error fingerprinting Table 197 ''public.PEOPLE''
clojure.lang.ExceptionInfo: Error executing query: ERROR: column PEOPLE.created_at does not exist
  Position: 20 {:driver :postgres, :sql ["-- Metabase" "SELECT" "  \"public\".\"PEOPLE\".\"created_at\" AS \"created_at\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"city\", 1, 1234) AS \"substring65957\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"state\", 1, 1234) AS \"substring65958\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"name\", 1, 1234) AS \"substring65959\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"address\", 1, 1234) AS \"substring65960\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"source\", 1, 1234) AS \"substring65961\"," "  \"public\".\"PEOPLE\".\"birth_date\" AS \"birth_date\"," "  \"public\".\"PEOPLE\".\"longitude\" AS \"longitude\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"password\", 1, 1234) AS \"substring65962\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"source2\", 1, 1234) AS \"substring65963\"," "  \"public\".\"PEOPLE\".\"latitude\" AS \"latitude\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"email\", 1, 1234) AS \"substring65964\"," "  SUBSTRING(\"public\".\"PEOPLE\".\"zip\", 1, 1234) AS \"substring65965\"" "FROM" "  \"public\".\"PEOPLE\"" "LIMIT" "  10000"], :params nil, :type :invalid-query}
	at metabase.driver.sql_jdbc.execute$execute_reducible_query$fn__80023$fn__80024.invoke(execute.clj:747)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query$fn__80023.invoke(execute.clj:744)
	at metabase.driver.sql_jdbc.execute$fn__79812$fn__79813.invoke(execute.clj:399)
	at metabase.driver.sql_jdbc.execute$do_with_resolved_connection79782__79783.invokeStatic(execute.clj:339)
	at metabase.driver.sql_jdbc.execute$do_with_resolved_connection79782__79783.invoke(execute.clj:322)
	at metabase.driver.sql_jdbc.execute$fn__79812.invokeStatic(execute.clj:393)
	at metabase.driver.sql_jdbc.execute$fn__79812.invoke(execute.clj:391)
	at clojure.lang.MultiFn.invoke(MultiFn.java:244)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:738)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:724)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:735)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:724)
	at metabase.driver.sql_jdbc$fn__134857.invokeStatic(sql_jdbc.clj:79)
	at metabase.driver.sql_jdbc$fn__134857.invoke(sql_jdbc.clj:77)
	at clojure.lang.MultiFn.invoke(MultiFn.java:244)
	at metabase.query_processor.pipeline$_STAR_execute_STAR_.invokeStatic(pipeline.clj:52)
	at metabase.query_processor.pipeline$_STAR_execute_STAR_.invoke(pipeline.clj:39)
	at metabase.query_processor.pipeline$_STAR_run_STAR_.invokeStatic(pipeline.clj:102)
	at metabase.query_processor.pipeline$_STAR_run_STAR_.invoke(pipeline.clj:95)
	at metabase.query_processor.execute$run.invokeStatic(execute.clj:62)
	at metabase.query_processor.execute$run.invoke(execute.clj:56)
	at metabase.query_processor.middleware.update_used_cards$update_used_cards_BANG_76369__76370$fn__76371.invoke(update_used_cards.clj:62)
	at metabase.query_processor.execute$add_native_form_to_result_metadata$fn__76474.invoke(execute.clj:25)
	at metabase.query_processor.execute$add_preprocessed_query_to_result_metadata_for_userland_query$fn__76480.invoke(execute.clj:36)
	at metabase.query_processor.middleware.cache$maybe_return_cached_results$maybe_return_cached_results_STAR___76317.invoke(cache.clj:241)
	at metabase.query_processor.middleware.permissions$check_query_permissions$fn__76083.invoke(permissions.clj:148)
	at metabase_enterprise.advanced_permissions.query_processor.middleware.permissions$fn__128255$check_download_permissions__128256$fn__128257.invoke(permissions.clj:99)
	at metabase.query_processor.middleware.enterprise$check_download_permissions_middleware$fn__76184.invoke(enterprise.clj:57)
	at metabase_enterprise.sandbox.query_processor.middleware.column_level_perms_check$fn__130429$maybe_apply_column_level_perms_check__130430$fn__130431.invoke(column_level_perms_check.clj:38)
	at metabase.query_processor.middleware.enterprise$maybe_apply_column_level_perms_check_middleware$fn__76194.invoke(enterprise.clj:70)
	at metabase.query_processor.execute$execute76507__76508$fn__76509.invoke(execute.clj:94)
	at metabase.query_processor.setup$do_with_qp_setup76460__76461.invokeStatic(setup.clj:225)
	at metabase.query_processor.setup$do_with_qp_setup76460__76461.invoke(setup.clj:216)
	at metabase.query_processor.execute$execute76507__76508.invokeStatic(execute.clj:93)
	at metabase.query_processor.execute$execute76507__76508.invoke(execute.clj:89)
	at metabase.query_processor$process_query_STAR__STAR_.invokeStatic(query_processor.clj:49)
	at metabase.query_processor$process_query_STAR__STAR_.invoke(query_processor.clj:44)
	at metabase_enterprise.audit_app.query_processor.middleware.handle_audit_queries$fn__82765$handle_audit_app_internal_queries__82766$fn__82767.invoke(handle_audit_queries.clj:143)
	at metabase.query_processor.middleware.enterprise$handle_audit_app_internal_queries_middleware$fn__76222.invoke(enterprise.clj:109)
	at metabase.query_processor.middleware.process_userland_query$process_userland_query_middleware82973__82974$fn__82975.invoke(process_userland_query.clj:208)
	at metabase.query_processor.middleware.catch_exceptions$catch_exceptions83038__83039$fn__83040.invoke(catch_exceptions.clj:128)
	at metabase.query_processor$process_query83201__83202$fn__83203.invoke(query_processor.clj:80)
	at metabase.query_processor.setup$do_with_canceled_chan76455__76456$fn__76457.invoke(setup.clj:189)
	at metabase.query_processor.setup$do_with_database_local_settings76448__76449$fn__76450.invoke(setup.clj:181)
	at metabase.query_processor.setup$do_with_driver76441__76442$fn__76443$fn__76444.invoke(setup.clj:166)
	at metabase.driver$do_with_driver.invokeStatic(driver.clj:112)
	at metabase.driver$do_with_driver.invoke(driver.clj:107)
	at metabase.query_processor.setup$do_with_driver76441__76442$fn__76443.invoke(setup.clj:165)
	at metabase.query_processor.setup$do_with_metadata_provider76432__76433$fn__76434.invoke(setup.clj:140)
	at metabase.query_processor.setup$do_with_resolved_database76422__76423$fn__76424.invoke(setup.clj:128)
	at metabase.query_processor.setup$do_with_qp_setup76460__76461.invokeStatic(setup.clj:232)
	at metabase.query_processor.setup$do_with_qp_setup76460__76461.invoke(setup.clj:216)
	at metabase.query_processor$process_query83201__83202.invokeStatic(query_processor.clj:78)
	at metabase.query_processor$process_query83201__83202.invoke(query_processor.clj:71)
	at metabase.db.metadata_queries$table_rows_sample83286__83287.invokeStatic(metadata_queries.clj:191)
	at metabase.db.metadata_queries$table_rows_sample83286__83287.invoke(metadata_queries.clj:174)
	at metabase.db.metadata_queries$fn__83289.invokeStatic(metadata_queries.clj:195)
	at metabase.db.metadata_queries$fn__83289.invoke(metadata_queries.clj:193)
	at clojure.lang.MultiFn.invoke(MultiFn.java:252)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_BANG_92958__92959.invokeStatic(fingerprint.clj:88)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_BANG_92958__92959.invoke(fingerprint.clj:65)
	at metabase.sync.analyze.fingerprint$fingerprint_table_BANG_93007__93008$fn__93009.invoke(fingerprint.clj:205)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:191)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:184)
	at metabase.sync.analyze.fingerprint$fingerprint_table_BANG_93007__93008.invokeStatic(fingerprint.clj:203)
	at metabase.sync.analyze.fingerprint$fingerprint_table_BANG_93007__93008.invoke(fingerprint.clj:197)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016$fn__93017$fn__93018.invoke(fingerprint.clj:230)
	at clojure.core$completing$fn__8558.invoke(core.clj:7010)
	at clojure.core$map$fn__5950$fn__5951.invoke(core.clj:2759)
	at toucan2.pipeline$with_init$fn__23762.invoke(pipeline.clj:327)
	at clojure.core$completing$fn__8558.invoke(core.clj:7010)
	at clojure.core$map$fn__5950$fn__5951.invoke(core.clj:2759)
	at toucan2.jdbc.result_set$reduce_result_set.invokeStatic(result_set.clj:156)
	at toucan2.jdbc.result_set$reduce_result_set.invoke(result_set.clj:125)
	at toucan2.jdbc.query$reduce_jdbc_query.invokeStatic(query.clj:51)
	at toucan2.jdbc.query$reduce_jdbc_query.invoke(query.clj:22)
	at toucan2.jdbc.pipeline$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invokeStatic(pipeline.clj:19)
	at toucan2.jdbc.pipeline$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invoke(pipeline.clj:9)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:15)
	at methodical.impl.combo.threaded$combine_methods_thread_last$fn__20325$combined_method_thread_last__20326.invoke(threaded.clj:64)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:216)
	at toucan2.pipeline$transduce_execute$with_connection_STAR___23682.invoke(pipeline.clj:78)
	at toucan2.connection$bind_current_connectable_fn$fn__23359.invoke(connection.clj:104)
	at toucan2.connection$bind_current_connectable_fn$fn__23359.invoke(connection.clj:104)
	at toucan2.connection$bind_current_connectable_fn$fn__23359.invoke(connection.clj:104)
	at toucan2.jdbc.connection$do_with_connection_primary_method_javax_sql_DataSource.invokeStatic(connection.clj:18)
	at toucan2.jdbc.connection$do_with_connection_primary_method_javax_sql_DataSource.invoke(connection.clj:15)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:210)
	at metabase.db.connection$do_with_connection_primary_method_default.invokeStatic(connection.clj:132)
	at metabase.db.connection$do_with_connection_primary_method_default.invoke(connection.clj:130)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:210)
	at toucan2.connection$do_with_connection_primary_method_.invokeStatic(connection.clj:204)
	at toucan2.connection$do_with_connection_primary_method_.invoke(connection.clj:194)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:12)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:210)
	at toucan2.pipeline$transduce_execute.invokeStatic(pipeline.clj:77)
	at toucan2.pipeline$transduce_execute.invoke(pipeline.clj:64)
	at clojure.lang.Var.invoke(Var.java:401)
	at toucan2.pipeline$transduce_compiled_query.invokeStatic(pipeline.clj:244)
	at toucan2.pipeline$transduce_compiled_query.invoke(pipeline.clj:240)
	at toucan2.pipeline$transduce_built_query.invokeStatic(pipeline.clj:252)
	at toucan2.pipeline$transduce_built_query.invoke(pipeline.clj:246)
	at toucan2.pipeline$transduce_query_primary_method_default.invokeStatic(pipeline.clj:272)
	at toucan2.pipeline$transduce_query_primary_method_default.invoke(pipeline.clj:269)
	at methodical.impl.combo.common$partial_STAR_$fn__20015.invoke(common.clj:15)
	at methodical.impl.combo.threaded$combine_methods_thread_last$fn__20325$combined_method_thread_last__20326.invoke(threaded.clj:64)
	at methodical.util.FnWithMeta.invoke(util.clj:46)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:216)
	at toucan2.pipeline$transduce_query_STAR_.invokeStatic(pipeline.clj:278)
	at toucan2.pipeline$transduce_query_STAR_.invoke(pipeline.clj:274)
	at toucan2.pipeline$transduce_with_model.invokeStatic(pipeline.clj:293)
	at toucan2.pipeline$transduce_with_model.invoke(pipeline.clj:280)
	at toucan2.pipeline$transduce_parsed.invokeStatic(pipeline.clj:309)
	at toucan2.pipeline$transduce_parsed.invoke(pipeline.clj:295)
	at toucan2.pipeline$transduce_unparsed.invokeStatic(pipeline.clj:317)
	at toucan2.pipeline$transduce_unparsed.invoke(pipeline.clj:311)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:669)
	at clojure.core$apply.invoke(core.clj:662)
	at toucan2.pipeline$reducible_fn$reify__23785.reduce(pipeline.clj:395)
	at clojure.core$transduce.invokeStatic(core.clj:7025)
	at clojure.core.Eduction.reduce(core.clj:7874)
	at clojure.core$reduce.invokeStatic(core.clj:6964)
	at clojure.core$reduce.invoke(core.clj:6947)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016$fn__93017.invoke(fingerprint.clj:228)
	at metabase.query_processor.store$do_with_metadata_provider56069__56070.invokeStatic(store.clj:170)
	at metabase.query_processor.store$do_with_metadata_provider56069__56070.invoke(store.clj:150)
	at metabase.query_processor.store$do_with_metadata_provider56069__56070.invokeStatic(store.clj:159)
	at metabase.query_processor.store$do_with_metadata_provider56069__56070.invoke(store.clj:150)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016.invokeStatic(fingerprint.clj:224)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016.invoke(fingerprint.clj:215)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016.invokeStatic(fingerprint.clj:219)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG__STAR_93015__93016.invoke(fingerprint.clj:215)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG_93023__93024.invokeStatic(fingerprint.clj:242)
	at metabase.sync.analyze.fingerprint$fingerprint_fields_for_db_BANG_93023__93024.invoke(fingerprint.clj:237)
	at metabase.sync.analyze$make_analyze_steps$fn__93174.invoke(analyze.clj:100)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:669)
	at clojure.core$apply.invoke(core.clj:662)
	at metabase.sync.util$run_step_with_metadata69879__69881$fn__69883$fn__69886.invoke(util.clj:495)
	at metabase.models.task_history$do_with_task_history69716__69717.invokeStatic(task_history.clj:117)
	at metabase.models.task_history$do_with_task_history69716__69717.invoke(task_history.clj:105)
	at metabase.sync.util$run_step_with_metadata69879__69881$fn__69883.doInvoke(util.clj:488)
	at clojure.lang.RestFn.invoke(RestFn.java:400)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:132)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:126)
	at metabase.sync.util$do_with_start_and_finish_debug_logging.invokeStatic(util.clj:150)
	at metabase.sync.util$do_with_start_and_finish_debug_logging.invoke(util.clj:146)
	at metabase.sync.util$run_step_with_metadata69879__69881.invokeStatic(util.clj:482)
	at metabase.sync.util$run_step_with_metadata69879__69881.invoke(util.clj:477)
	at metabase.sync.util$run_sync_operation69929__69930$fn__69931$fn__69939.invoke(util.clj:568)
	at metabase.sync.util$run_sync_operation69929__69930$fn__69931.invoke(util.clj:566)
	at metabase.models.task_history$do_with_task_history69716__69717.invokeStatic(task_history.clj:117)
	at metabase.models.task_history$do_with_task_history69716__69717.invoke(task_history.clj:105)
	at metabase.sync.util$run_sync_operation69929__69930.invokeStatic(util.clj:563)
	at metabase.sync.util$run_sync_operation69929__69930.invoke(util.clj:558)
	at metabase.sync.analyze$analyze_db_BANG_93181__93182$fn__93183.invoke(analyze.clj:116)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:191)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:184)
	at clojure.core$partial$fn__5929.invoke(core.clj:2647)
	at metabase.driver$fn__53400.invokeStatic(driver.clj:943)
	at metabase.driver$fn__53400.invoke(driver.clj:943)
	at clojure.lang.MultiFn.invoke(MultiFn.java:239)
	at metabase.sync.util$sync_in_context$fn__69788.invoke(util.clj:167)
	at metabase.sync.util$with_db_logging_disabled$fn__69785.invoke(util.clj:159)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:132)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:126)
	at metabase.sync.util$with_start_and_finish_logging$fn__69772.invoke(util.clj:144)
	at metabase.sync.util$with_sync_events69762__69763$fn__69767.invoke(util.clj:118)
	at metabase.sync.util$with_duplicate_ops_prevented$fn__69752.invoke(util.clj:90)
	at metabase.sync.util$do_sync_operation69803__69804.invokeStatic(util.clj:216)
	at metabase.sync.util$do_sync_operation69803__69804.invoke(util.clj:210)
	at metabase.sync.analyze$analyze_db_BANG_93181__93182.invokeStatic(analyze.clj:114)
	at metabase.sync.analyze$analyze_db_BANG_93181__93182.invoke(analyze.clj:109)
	at metabase.sync.sync$do_phase_BANG_.invokeStatic(sync.clj:49)
	at metabase.sync.sync$do_phase_BANG_.invoke(sync.clj:47)
	at clojure.core$partial$fn__5927.invoke(core.clj:2641)
	at clojure.core$keep$fn__8691.invoke(core.clj:7529)
	at clojure.lang.LazySeq.force(LazySeq.java:50)
	at clojure.lang.LazySeq.realize(LazySeq.java:89)
	at clojure.lang.LazySeq.seq(LazySeq.java:106)
	at clojure.lang.RT.seq(RT.java:555)
	at clojure.core$seq__5486.invokeStatic(core.clj:139)
	at clojure.core$dorun.invokeStatic(core.clj:3141)
	at clojure.core$doall.invokeStatic(core.clj:3156)
	at clojure.core$doall.invoke(core.clj:3156)
	at metabase.sync.sync$sync_database_BANG_93328__93330$fn__93332.invoke(sync.clj:74)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:191)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:184)
	at clojure.core$partial$fn__5929.invoke(core.clj:2647)
	at metabase.driver$fn__53400.invokeStatic(driver.clj:943)
	at metabase.driver$fn__53400.invoke(driver.clj:943)
	at clojure.lang.MultiFn.invoke(MultiFn.java:239)
	at metabase.sync.util$sync_in_context$fn__69788.invoke(util.clj:167)
	at metabase.sync.util$with_db_logging_disabled$fn__69785.invoke(util.clj:159)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:132)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:126)
	at metabase.sync.util$with_start_and_finish_logging$fn__69772.invoke(util.clj:144)
	at metabase.sync.util$with_sync_events69762__69763$fn__69767.invoke(util.clj:118)
	at metabase.sync.util$with_duplicate_ops_prevented$fn__69752.invoke(util.clj:90)
	at metabase.sync.util$do_sync_operation69803__69804.invokeStatic(util.clj:216)
	at metabase.sync.util$do_sync_operation69803__69804.invoke(util.clj:210)
	at metabase.sync.sync$sync_database_BANG_93328__93330.invokeStatic(sync.clj:71)
	at metabase.sync.sync$sync_database_BANG_93328__93330.invoke(sync.clj:57)
	at metabase.sync.sync$sync_database_BANG_93328__93330.invokeStatic(sync.clj:66)
	at metabase.sync.sync$sync_database_BANG_93328__93330.invoke(sync.clj:57)
	at metabase.sync.events.sync_database$publish_event_BANG__primary_method_metabase_sync_events_sync_database_event$fn__129528.invoke(sync_database.clj:26)
	at metabase.util.quick_task$submit_task_BANG_$fn__80172.invoke(quick_task.clj:27)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$with_bindings_STAR_.invokeStatic(core.clj:1990)
	at clojure.core$with_bindings_STAR_.doInvoke(core.clj:1990)
	at clojure.lang.RestFn.invoke(RestFn.java:428)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:135)
	at clojure.core$apply.invokeStatic(core.clj:671)
	at clojure.core$bound_fn_STAR_$fn__5837.doInvoke(core.clj:2020)
	at clojure.lang.RestFn.invoke(RestFn.java:400)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.postgresql.util.PSQLException: ERROR: column PEOPLE.created_at does not exist
  Position: 20
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2733)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2420)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:372)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:517)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:434)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:356)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:341)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:317)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:312)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.execute(NewProxyStatement.java:1341)
	at metabase.driver.sql_jdbc.execute$fn__79936.invokeStatic(execute.clj:571)
	at metabase.driver.sql_jdbc.execute$fn__79936.invoke(execute.clj:569)
	at clojure.lang.MultiFn.invoke(MultiFn.java:239)
	at metabase.driver.sql_jdbc.execute$execute_statement_or_prepared_statement_BANG_.invokeStatic(execute.clj:579)
	at metabase.driver.sql_jdbc.execute$execute_statement_or_prepared_statement_BANG_.invoke(execute.clj:576)
	at metabase.driver.sql_jdbc.execute$execute_reducible_query$fn__80023$fn__80024.invoke(execute.clj:745)
	... 253 more
2025-04-20 23:16:09,821 INFO analyze.fingerprint :: Fingerprinting 5 fields in table Table 198 ''public.reviews''
2025-04-20 23:16:09,850 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2952 ''product_id''
2025-04-20 23:16:09,856 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2953 ''body''
2025-04-20 23:16:09,860 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2954 ''rating''
2025-04-20 23:16:09,866 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2955 ''created_at''
2025-04-20 23:16:09,871 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2957 ''reviewer''
2025-04-20 23:16:09,904 INFO sync.analyze :: fingerprint-fields Analyzed [**********········································]    21% Table 200 ''public.invoices''
2025-04-20 23:16:09,907 INFO analyze.fingerprint :: Fingerprinting 5 fields in table Table 200 ''public.invoices''
2025-04-20 23:16:10,013 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2921 ''expected_invoice''
2025-04-20 23:16:10,018 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2916 ''payment''
2025-04-20 23:16:10,024 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2918 ''date_received''
2025-04-20 23:16:10,028 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2920 ''plan''
2025-04-20 23:16:10,034 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2917 ''account_id''
2025-04-20 23:16:10,039 INFO analyze.fingerprint :: Fingerprinting 8 fields in table Table 199 ''public.orders''
2025-04-20 23:16:10,161 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2922 ''subtotal''
2025-04-20 23:16:10,167 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2925 ''tax''
2025-04-20 23:16:10,171 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2926 ''created_at''
2025-04-20 23:16:10,174 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2927 ''quantity''
2025-04-20 23:16:10,177 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2928 ''discount''
2025-04-20 23:16:10,181 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2930 ''total''
2025-04-20 23:16:10,184 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2923 ''product_id''
2025-04-20 23:16:10,187 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2924 ''user_id''
2025-04-20 23:16:10,194 INFO sync.analyze :: fingerprint-fields Analyzed [****************··································]    32% Table 202 ''public.analytic_events''
2025-04-20 23:16:10,199 INFO analyze.fingerprint :: Fingerprinting 5 fields in table Table 202 ''public.analytic_events''
2025-04-20 23:16:10,251 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2903 ''event''
2025-04-20 23:16:10,255 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2904 ''timestamp''
2025-04-20 23:16:10,255 DEBUG middleware.log :: GET /api/health 200 0ms (0 DB calls) App DB connections: 1/13 Jetty threads: 6/50 (2 idle, 0 queued) (131 total active threads) Queries in flight: 0 (0 queued) {:metabase-user-id nil}
2025-04-20 23:16:10,258 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2905 ''page_url''
2025-04-20 23:16:10,262 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2907 ''button_label''
2025-04-20 23:16:10,266 DEBUG analyze.fingerprint :: Saving fingerprint for Field 2906 ''account_id''
2025-04-20 23:16:10,273 INFO sync.util :: FINISHED: step ''fingerprint-fields'' for postgres Database 13371341 ''data'' (919.8 ms)
2025-04-20 23:16:10,275 INFO sync.util :: STARTING: step ''classify-fields'' for postgres Database 13371341 ''data''
2025-04-20 23:16:10,283 DEBUG analyze.classify :: Based on classification, updating these values of Field 2887 ''latitude'': {:semantic_type :type/Latitude}
2025-04-20 23:16:10,287 DEBUG analyze.classify :: Based on classification, updating these values of Field 2888 ''country'': {:semantic_type :type/Country}
2025-04-20 23:16:10,290 DEBUG analyze.classify :: Based on classification, updating these values of Field 2889 ''email'': {:semantic_type :type/Email}
2025-04-20 23:16:10,294 DEBUG analyze.classify :: Based on classification, updating these values of Field 2891 ''longitude'': {:semantic_type :type/Longitude}
2025-04-20 23:16:10,298 DEBUG analyze.classify :: Based on classification, updating these values of Field 2892 ''trial_converted'': {:semantic_type :type/Category}
2025-04-20 23:16:10,302 DEBUG analyze.classify :: Based on classification, updating these values of Field 2893 ''plan'': {:semantic_type :type/Category}
2025-04-20 23:16:10,306 DEBUG analyze.classify :: Based on classification, updating these values of Field 2896 ''last_name'': {:semantic_type :type/Name}
2025-04-20 23:16:10,310 DEBUG analyze.classify :: Based on classification, updating these values of Field 2897 ''source'': {:semantic_type :type/Source}
2025-04-20 23:16:10,315 DEBUG analyze.classify :: Based on classification, updating these values of Field 2898 ''legacy_plan'': {:semantic_type :type/Category}
2025-04-20 23:16:10,319 DEBUG analyze.classify :: Based on classification, updating these values of Field 2899 ''canceled_at'': {:semantic_type :type/CancelationTimestamp}
2025-04-20 23:16:10,322 DEBUG analyze.classify :: Based on classification, updating these values of Field 2900 ''first_name'': {:semantic_type :type/Name}
2025-04-20 23:16:10,326 DEBUG analyze.classify :: Based on classification, updating these values of Field 2901 ''active_subscription'': {:semantic_type :type/Category}
2025-04-20 23:16:10,330 DEBUG analyze.classify :: Based on classification, updating these values of Field 2902 ''created_at'': {:semantic_type :type/CreationTimestamp}
2025-04-20 23:16:10,334 DEBUG analyze.classify :: Based on classification, updating these values of Field 2909 ''email'': {:semantic_type :type/Email}
2025-04-20 23:16:10,338 DEBUG analyze.classify :: Based on classification, updating these values of Field 2910 ''rating_mapped'': {:semantic_type :type/Category}
2025-04-20 23:16:10,341 DEBUG analyze.classify :: Based on classification, updating these values of Field 2911 ''rating'': {:semantic_type :type/Score}
2025-04-20 23:16:10,346 DEBUG analyze.classify :: Based on classification, updating these values of Field 2914 ''body'': {:preview_display false}
2025-04-20 23:16:10,351 DEBUG analyze.classify :: Based on classification, updating these values of Field 2944 ''category'': {:semantic_type :type/Category}
2025-04-20 23:16:10,354 DEBUG analyze.classify :: Based on classification, updating these values of Field 2947 ''created_at'': {:semantic_type :type/CreationTimestamp}
2025-04-20 23:16:10,357 DEBUG analyze.classify :: Based on classification, updating these values of Field 2948 ''vendor'': {:semantic_type :type/Company}
2025-04-20 23:16:10,360 DEBUG analyze.classify :: Based on classification, updating these values of Field 2949 ''title'': {:semantic_type :type/Title}
2025-04-20 23:16:10,366 DEBUG analyze.classify :: Based on classification, updating these values of Field 2950 ''rating'': {:semantic_type :type/Score}
2025-04-20 23:16:10,369 INFO sync.analyze :: classify-fields Analyzed [*********************·····························]    43% Table 196 ''public.products''
2025-04-20 23:16:10,373 DEBUG analyze.classify :: Based on classification, updating these values of Field 2953 ''body'': {:preview_display false}
2025-04-20 23:16:10,377 DEBUG analyze.classify :: Based on classification, updating these values of Field 2954 ''rating'': {:semantic_type :type/Score}
2025-04-20 23:16:10,383 DEBUG analyze.classify :: Based on classification, updating these values of Field 2955 ''created_at'': {:semantic_type :type/CreationTimestamp}
2025-04-20 23:16:10,387 DEBUG analyze.classify :: Based on classification, updating these values of Field 2921 ''expected_invoice'': {:semantic_type :type/Category}
2025-04-20 23:16:10,391 DEBUG analyze.classify :: Based on classification, updating these values of Field 2920 ''plan'': {:semantic_type :type/Category}
2025-04-20 23:16:10,394 INFO sync.analyze :: classify-fields Analyzed [**************************························]    54% Table 200 ''public.invoices''
2025-04-20 23:16:10,399 DEBUG analyze.classify :: Based on classification, updating these values of Field 2926 ''created_at'': {:semantic_type :type/CreationTimestamp}
2025-04-20 23:16:10,403 DEBUG analyze.classify :: Based on classification, updating these values of Field 2927 ''quantity'': {:semantic_type :type/Quantity}
2025-04-20 23:16:10,407 DEBUG analyze.classify :: Based on classification, updating these values of Field 2928 ''discount'': {:semantic_type :type/Discount}
2025-04-20 23:16:10,415 DEBUG analyze.classify :: Based on classification, updating these values of Field 2903 ''event'': {:semantic_type :type/Category}
2025-04-20 23:16:10,418 DEBUG analyze.classify :: Based on classification, updating these values of Field 2905 ''page_url'': {:semantic_type :type/URL}
2025-04-20 23:16:10,422 DEBUG analyze.classify :: Based on classification, updating these values of Field 2907 ''button_label'': {:semantic_type :type/Category}
2025-04-20 23:16:10,426 INFO sync.analyze :: classify-fields Analyzed [********************************··················]    64% Table 202 ''public.analytic_events''
2025-04-20 23:16:10,432 INFO sync.util :: FINISHED: step ''classify-fields'' for postgres Database 13371341 ''data'' (156.1 ms)
2025-04-20 23:16:10,434 INFO sync.util :: STARTING: step ''classify-tables'' for postgres Database 13371341 ''data''
2025-04-20 23:16:10,438 DEBUG analyze.classify :: Based on classification, updating these values of Table 203 ''public.accounts'': {:entity_type :entity/UserTable}
2025-04-20 23:16:10,446 DEBUG analyze.classify :: Based on classification, updating these values of Table 204 ''public.feedback'': {:entity_type :entity/GenericTable}
2025-04-20 23:16:10,452 DEBUG analyze.classify :: Based on classification, updating these values of Table 196 ''public.products'': {:entity_type :entity/ProductTable}
2025-04-20 23:16:10,457 INFO sync.analyze :: classify-tables Analyzed [*************************************·············]    75% Table 196 ''public.products''
2025-04-20 23:16:10,458 DEBUG analyze.classify :: Based on classification, updating these values of Table 197 ''public.PEOPLE'': {:entity_type :entity/UserTable}
2025-04-20 23:16:10,463 DEBUG analyze.classify :: Based on classification, updating these values of Table 198 ''public.reviews'': {:entity_type :entity/GenericTable}
2025-04-20 23:16:10,468 DEBUG analyze.classify :: Based on classification, updating these values of Table 200 ''public.invoices'': {:entity_type :entity/GenericTable}
2025-04-20 23:16:10,473 INFO sync.analyze :: classify-tables Analyzed [******************************************········]    86% Table 200 ''public.invoices''
2025-04-20 23:16:10,475 DEBUG analyze.classify :: Based on classification, updating these values of Table 199 ''public.orders'': {:entity_type :entity/TransactionTable}
2025-04-20 23:16:10,482 DEBUG analyze.classify :: Based on classification, updating these values of Table 201 ''public.people'': {:entity_type :entity/UserTable}
2025-04-20 23:16:10,486 DEBUG analyze.classify :: Based on classification, updating these values of Table 202 ''public.analytic_events'': {:entity_type :entity/EventTable}
2025-04-20 23:16:10,491 INFO sync.analyze :: classify-tables Analyzed [************************************************··]    96% Table 202 ''public.analytic_events''
2025-04-20 23:16:10,496 INFO sync.util :: FINISHED: step ''classify-tables'' for postgres Database 13371341 ''data'' (62.6 ms)
2025-04-20 23:16:10,499 DEBUG sync.util :: 
#################################################################
# Completed analyze on data
# Start: 2025-04-20T23:16:09.353370602Z
# End: 2025-04-20T23:16:10.49963258Z
# Duration: 1.1 s
# ---------------------------------------------------------------
# Completed step ''fingerprint-fields''
# Start: 2025-04-20T23:16:09.353382935Z
# End: 2025-04-20T23:16:10.275865053Z
# Duration: 922.5 ms
# Fingerprint updates attempted 51, updated 51, no data found 0, failed 0
# ---------------------------------------------------------------
# Completed step ''classify-fields''
# Start: 2025-04-20T23:16:10.275928773Z
# End: 2025-04-20T23:16:10.434152045Z
# Duration: 158.2 ms
# Total number of fields classified 51, 0 failed
# ---------------------------------------------------------------
# Completed step ''classify-tables''
# Start: 2025-04-20T23:16:10.434168596Z
# End: 2025-04-20T23:16:10.499612222Z
# Duration: 65.4 ms
# Total number of tables classified 9, 9 updated
#################################################################

2025-04-20 23:16:10,508 INFO sync.util :: FINISHED: Analyze data for postgres Database 13371341 ''data'' (1.2 s)
2025-04-20 23:16:10,510 DEBUG sync.util :: Sync operations in flight: {:cache-field-values #{13371341}, :sync #{13371341}}
2025-04-20 23:16:10,512 INFO sync.util :: STARTING: Cache field values in postgres Database 13371341 ''data''
2025-04-20 23:16:10,516 INFO sync.util :: STARTING: step ''delete-expired-advanced-field-values'' for postgres Database 13371341 ''data''
2025-04-20 23:16:10,537 DEBUG middleware.log :: GET /api/database/13371341 200 2ms (1 DB calls) App DB connections: 1/13 Jetty threads: 6/50 (2 idle, 0 queued) (131 total active threads) Queries in flight: 0 (0 queued) {:metabase-user-id 1}
2025-04-20 23:16:10,601 INFO sync.util :: FINISHED: step ''delete-expired-advanced-field-values'' for postgres Database 13371341 ''data'' (85.5 ms)
2025-04-20 23:16:10,604 INFO sync.util :: STARTING: step ''update-field-values'' for postgres Database 13371341 ''data''
2025-04-20 23:16:10,611 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2900 ''first_name''
2025-04-20 23:16:10,615 INFO sync.field-values :: Field Field 2900 ''first_name'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,617 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2888 ''country''
2025-04-20 23:16:10,619 INFO sync.field-values :: Field Field 2888 ''country'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,621 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2889 ''email''
2025-04-20 23:16:10,623 INFO sync.field-values :: Field Field 2889 ''email'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,625 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2892 ''trial_converted''
2025-04-20 23:16:10,627 INFO sync.field-values :: Field Field 2892 ''trial_converted'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,628 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2893 ''plan''
2025-04-20 23:16:10,631 INFO sync.field-values :: Field Field 2893 ''plan'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,634 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2896 ''last_name''
2025-04-20 23:16:10,637 INFO sync.field-values :: Field Field 2896 ''last_name'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,639 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2897 ''source''
2025-04-20 23:16:10,641 INFO sync.field-values :: Field Field 2897 ''source'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,644 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2895 ''seats''
2025-04-20 23:16:10,646 INFO sync.field-values :: Field Field 2895 ''seats'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,648 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2901 ''active_subscription''
2025-04-20 23:16:10,650 INFO sync.field-values :: Field Field 2901 ''active_subscription'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,652 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2898 ''legacy_plan''
2025-04-20 23:16:10,654 INFO sync.field-values :: Field Field 2898 ''legacy_plan'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,657 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2909 ''email''
2025-04-20 23:16:10,659 INFO sync.field-values :: Field Field 2909 ''email'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,660 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2910 ''rating_mapped''
2025-04-20 23:16:10,663 INFO sync.field-values :: Field Field 2910 ''rating_mapped'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,664 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2911 ''rating''
2025-04-20 23:16:10,668 INFO sync.field-values :: Field Field 2911 ''rating'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,670 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2914 ''body''
2025-04-20 23:16:10,672 INFO sync.field-values :: Field Field 2914 ''body'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,676 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2948 ''vendor''
2025-04-20 23:16:10,678 INFO sync.field-values :: Field Field 2948 ''vendor'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,680 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2949 ''title''
2025-04-20 23:16:10,681 INFO sync.field-values :: Field Field 2949 ''title'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,684 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2944 ''category''
2025-04-20 23:16:10,686 INFO sync.field-values :: Field Field 2944 ''category'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,688 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2945 ''ean''
2025-04-20 23:16:10,690 INFO sync.field-values :: Field Field 2945 ''ean'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,693 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2932 ''city''
2025-04-20 23:16:10,695 INFO sync.field-values :: Field Field 2932 ''city'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,697 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2933 ''state''
2025-04-20 23:16:10,700 INFO sync.field-values :: Field Field 2933 ''state'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,702 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2934 ''name''
2025-04-20 23:16:10,704 INFO sync.field-values :: Field Field 2934 ''name'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,706 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2935 ''address''
2025-04-20 23:16:10,708 INFO sync.field-values :: Field Field 2935 ''address'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,710 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2936 ''source''
2025-04-20 23:16:10,712 INFO sync.field-values :: Field Field 2936 ''source'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,714 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2886 ''id''
2025-04-20 23:16:10,716 INFO sync.field-values :: Field Field 2886 ''id'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,719 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2939 ''password''
2025-04-20 23:16:10,721 INFO sync.field-values :: Field Field 2939 ''password'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,722 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2940 ''source2''
2025-04-20 23:16:10,724 INFO sync.field-values :: Field Field 2940 ''source2'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,726 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2942 ''email''
2025-04-20 23:16:10,728 INFO sync.field-values :: Field Field 2942 ''email'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,729 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2943 ''zip''
2025-04-20 23:16:10,732 INFO sync.field-values :: Field Field 2943 ''zip'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,735 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2954 ''rating''
2025-04-20 23:16:10,737 INFO sync.field-values :: Field Field 2954 ''rating'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,739 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2957 ''reviewer''
2025-04-20 23:16:10,741 INFO sync.field-values :: Field Field 2957 ''reviewer'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,743 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2953 ''body''
2025-04-20 23:16:10,745 INFO sync.field-values :: Field Field 2953 ''body'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,748 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2921 ''expected_invoice''
2025-04-20 23:16:10,750 INFO sync.field-values :: Field Field 2921 ''expected_invoice'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,752 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2920 ''plan''
2025-04-20 23:16:10,754 INFO sync.field-values :: Field Field 2920 ''plan'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,758 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2927 ''quantity''
2025-04-20 23:16:10,761 INFO sync.field-values :: Field Field 2927 ''quantity'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,765 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2907 ''button_label''
2025-04-20 23:16:10,768 INFO sync.field-values :: Field Field 2907 ''button_label'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,770 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2903 ''event''
2025-04-20 23:16:10,772 INFO sync.field-values :: Field Field 2903 ''event'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,774 DEBUG sync.field-values :: Looking into updating FieldValues for Field 2905 ''page_url''
2025-04-20 23:16:10,776 INFO sync.field-values :: Field Field 2905 ''page_url'' does not have FieldValues. Skipping...
2025-04-20 23:16:10,781 INFO sync.util :: FINISHED: step ''update-field-values'' for postgres Database 13371341 ''data'' (177.5 ms)
2025-04-20 23:16:10,784 DEBUG sync.util :: 
#################################################################
# Completed field values scanning on data
# Start: 2025-04-20T23:16:10.516035105Z
# End: 2025-04-20T23:16:10.78386792Z
# Duration: 267.8 ms
# ---------------------------------------------------------------
# Completed step ''delete-expired-advanced-field-values''
# Start: 2025-04-20T23:16:10.516040115Z
# End: 2025-04-20T23:16:10.604123062Z
# Duration: 88.1 ms
# Deleted 0 expired advanced fieldvalues
# ---------------------------------------------------------------
# Completed step ''update-field-values''
# Start: 2025-04-20T23:16:10.604151485Z
# End: 2025-04-20T23:16:10.783853674Z
# Duration: 179.7 ms
# Updated 0 field value sets, created 0, deleted 0 with 0 errors
#################################################################

2025-04-20 23:16:10,787 INFO sync.util :: FINISHED: Cache field values in postgres Database 13371341 ''data'' (274.9 ms)
2025-04-20 23:16:10,790 INFO sync.util :: FINISHED: Sync postgres Database 13371341 ''data'' (2.4 s)
```

### Information about your Metabase installation

```JSON
v54.x, but I would guess that this comes from before, I'm not very sure since we changed the sync process several times
```

### Severity

P2

### Additional context

We need to check which DW's support giving tables/schemas same name but with different capitalization as this might have a bigger impact.

--- a/src/metabase/sync/sync_metadata/fields.clj
+++ b/src/metabase/sync/sync_metadata/fields.clj
@@ -79,16 +79,23 @@
           schemas?        (driver.u/supports? driver :schemas database)
           fields-metadata (if schemas?
                             (fetch-metadata/fields-metadata database :schema-names (sync-util/sync-schemas database))
-                            (fetch-metadata/fields-metadata database))]
+                            (fetch-metadata/fields-metadata database))
+          select-best-matching-name (fn [target-schema target-name]
+                                      (fn [item]
+                                        [(not= (:schema item) target-schema)
+                                         (not= (:name item) target-name)]))]
       (transduce (comp
                   (partition-by (juxt :table-name :table-schema))
                   (map (fn [table-metadata]
-                         (let [fst     (first table-metadata)
-                               table   (t2/select-one :model/Table
-                                                      :db_id (:id database)
-                                                      :%lower.name (t2.util/lower-case-en (:table-name fst))
-                                                      :%lower.schema (some-> fst :table-schema t2.util/lower-case-en)
-                                                      {:where sync-util/sync-tables-clause})
+                         (let [{:keys [table-name table-schema]} (first table-metadata)
+
+                               table   (->> (t2/select :model/Table
+                                                       :db_id (:id database)
+                                                       :%lower.name (t2.util/lower-case-en table-name)
+                                                       :%lower.schema (some-> table-schema t2.util/lower-case-en)
+                                                       {:where sync-util/sync-tables-clause})
+                                            (sort-by (select-best-matching-name table-schema table-name))
+                                            first)
                                updated (if table
                                          (try
                                            ;; TODO: decouple nested field columns sync from field sync. This will allow

--- a/src/metabase/sync/sync_metadata/fields.clj
+++ b/src/metabase/sync/sync_metadata/fields.clj
@@ -69,6 +69,14 @@
      ;; `sync-instances`
      (sync-metadata/update-metadata! database table db-metadata (fields.our-metadata/our-metadata table))))
 
+(defn- select-best-matching-name
+  "Returns a key function for use with [[sort-by]] that ranks items based on how closely their `:schema` and `:name` match the given target values.
+   Items with matching `:schema` and `:name` are prioritized, with exact matches ranked higher than non-exact matches."
+  [target-schema target-name]
+  (fn [item]
+    [(not= (:schema item) target-schema)
+     (not= (:name item) target-name)]))
+
 (mu/defn sync-fields! :- [:map
                           [:updated-fields ms/IntGreaterThanOrEqualToZero]
                           [:total-fields   ms/IntGreaterThanOrEqualToZero]]
@@ -79,11 +87,7 @@
           schemas?        (driver.u/supports? driver :schemas database)
           fields-metadata (if schemas?
                             (fetch-metadata/fields-metadata database :schema-names (sync-util/sync-schemas database))
-                            (fetch-metadata/fields-metadata database))
-          select-best-matching-name (fn [target-schema target-name]
-                                      (fn [item]
-                                        [(not= (:schema item) target-schema)
-                                         (not= (:name item) target-name)]))]
+                            (fetch-metadata/fields-metadata database))]
       (transduce (comp
                   (partition-by (juxt :table-name :table-schema))
                   (map (fn [table-metadata]

--- a/test/metabase/sync/sync_metadata/fields_test.clj
+++ b/test/metabase/sync/sync_metadata/fields_test.clj
@@ -300,6 +300,29 @@
                                  (m/find-first (comp #{"metadata"} :name))
                                  :steps
                                  (m/find-first (comp #{"sync-fields"} first)))]
+        (is (=? ["sync-fields" {:total-fields 2 :updated-fields 2}] field-sync-info)))))
+
+  (testing "Two tables with same lower-case name can be synced (SEM-258)"
+    (one-off-dbs/with-blank-db
+      (doseq [statement [;; H2 needs that 'guest' user for QP purposes. Set that up
+                         "CREATE USER IF NOT EXISTS GUEST PASSWORD 'guest';"
+                         ;; Keep DB open until we say otherwise :)
+                         "SET DB_CLOSE_DELAY -1;"
+                         ;; create table & load data
+                         "DROP TABLE IF EXISTS \"birds\";"
+                         "DROP TABLE IF EXISTS \"BIRDS\";"
+                         "CREATE TABLE \"birds\" (\"event\" VARCHAR);"
+                         "CREATE TABLE \"BIRDS\" (\"event\" VARCHAR);"
+                         "GRANT ALL ON \"birds\" TO GUEST;"
+                         "GRANT ALL ON \"BIRDS\" TO GUEST;"
+                         "INSERT INTO \"birds\" (\"event\") VALUES ('a'), ('b')"
+                         "INSERT INTO \"BIRDS\" (\"event\") VALUES ('c'), ('d')"]]
+        (jdbc/execute! one-off-dbs/*conn* [statement]))
+      (let [sync-info (sync/sync-database! (mt/db))
+            field-sync-info (->> sync-info
+                                 (m/find-first (comp #{"metadata"} :name))
+                                 :steps
+                                 (m/find-first (comp #{"sync-fields"} first)))]
         (is (=? ["sync-fields" {:total-fields 2 :updated-fields 2}] field-sync-info))))))
 
 (mt/defdataset country

